### PR TITLE
changed website search to also match parts of words

### PR DESCRIPTION
**Pull request by danrot**
Merge with: https://github.com/sulu-io/sulu/pull/1617

__tasks:__

- [ ] test coverage
- [ ] gather feedback for my changes

__informations:__

| q                | a
| ---------------- | ---
| Fixed tickets    | none
| BC breaks        | none
| Documentation PR | none

### CODE REVIEW

**Comment by danrot** [comment]
Looks like the behat tests have to be fixed because of the redesign...

File: composer.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
         "sensio/distribution-bundle": "~3.0",
         "sensio/framework-extra-bundle": "~3.0",
         "incenteev/composer-parameter-handler": "~2.0",
-        "sulu/sulu": "dev-develop",
+        "sulu/sulu": "dev-bugfix/behat",
```
Comment (danrot) [inline composer.json]: change before merging

**Comment by wachterjohannes** [comment]
i have a page with title "Test-1" if i search for "Tes" i get the site but not for "est". is this the right behavior?

**Comment by danrot** [comment]
Yeah, at least for the default search we said that the result should begin with the query, because that's what people usually look for.

**Pull request merged**